modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -1,14 +1,49 @@
-[id="anchor-and-file-names"]
-= Anchor Names and File Names
+[id="module_anchor-and-file-names-concept"]
```

Comment:
Should this be con_anchor-and-file-names and the file also renamed to con_anchor-and-file-names.adoc to keep it consistent with the guideline that it is prescribing?

Further, we should add a guideline that terms like 'and', 'the' should not be put into filenames. Otherwise with this recommendation of keeping filenames the same as titles, we will end up with really long filenames like: https://github.com/openshift/openshift-docs/blob/master/modules/developer-cli-odo-creating-and-deploying-a-nodejs-application-with-odo.adoc

Reply:
Yes, this entire repository needs to be updated to reflect these recommendations and this will be done is a separate PR. 
We do not recommend that file or anchor names match the module headings exactly. Some groups will choose to do that, however.

Reply:
@vikram-redhat, I don't agree that words like 'and' or 'the' should not be in file names. Long file names are not good practice, I agree, but so are not long headings in general. Writers should -- wherever possible -- avoid long headings (which in turn eliminates long file names). Omitting 'and' or 'the' does not help that much in this regard.

More importantly, we have tooling (such as `newdoc` or `nebel`) for automatic generation of module and assembly files (pre-filled templates). These tools automate much of the tedious manual work a writer would need to do when starting on new docs. While it would be possible to modify the tools to omit a defined list of words from file names, it seems like an unnecessary exercise.

Reply:
@rkratky Ok and that is fine. So the guidance (or a hard requirement?) is that the file names/anchor names/module names must match exactly, excluding the "proc|con|ref" bit?

Reply:
@rkratky  @vikram-redhat  the guidance is that it is preferable (not required) for the anchor to match the file name exactly, including the prefix, and that the file name/anchor should approximate the actual module title.  However, I believe RHEL uses a tool that creates module IDs and file names from the module title so they include the small words and they follow a process of ensuring that the module title and anchor match exactly even when the module title changes. That is fine, but it's not what the board agreed to recommend. There are other variations on file naming that don't break Pantheon and unless these variations cause a Pantheon or other problem (ultimate single repository?) groups are free to deviate from the recommended formats as long as they include the prefix or the variable in each module.

Reply:
@ emmurphy1 right so this goes back to @rkratky's point about this being a RH specific guideline then if we are only including it for accomodating Pantheon and the RHEL tool.